scripts/manifest.js

```diff
@@ -44,10 +45,19 @@ const overrides = new Map([
 
 const icons = manifest.map(icon => {
 	const id = typeof icon === 'string' ? icon : icon.symbol;
+	const fileName = `${id.toLowerCase()}.svg`;
+	const svgPath = path.resolve(__dirname, '../svg/color/', fileName);
+	let color;
+	if (fs.existsSync(svgPath)) {
+		const svg = fs.readFileSync(svgPath, 'utf8');
+		const fillColor = getColors(svg).fills[0];
+		color = fillColor ? fillColor.hex().toUpperCase() : undefined;
```

Comment:
Same here, we should fail loudly (meaning throw an error) if the color cannot be found, as that's definitely something we'll want to look into.